README.md

```diff
@@ -278,11 +278,38 @@ discuss the performance and limitations for the two tasks
 
 ## Cite
 
-When referencing the data or the models provided, please cite this paper:
-
-*The manuscript is not ready yet. As soon as it is done, we will update this section and advertised it in the
-[ News & Announcements forum](https://www.iarai.ac.at/traffic4cast/forums/forum/news-announcements/), please subscribe!*
+When referencing the data or the models provided, please cite the paper below or this repository/
```

Comment:
@moritz9 For 2021, shouldn't we cite this repository only?

Reply:
Added the citation references for the 2019 and 2020 competition.